src/main/resources/assets/lambdynlights/lang/it_it.json

```diff
@@ -0,0 +1,19 @@
+{
+  "lambdynlights.menu.light_sources": "Fonti di Luci Dinamiche",
+  "lambdynlights.menu.title": "Impostazioni di LambDynamicLights",
+  "lambdynlights.menu.canvas.1": "Disattivare luci dinamiche di entità è raccomandato con Canvas!",
+  "lambdynlights.menu.canvas.2": "L'illuminazione dinamica con Canvas è pesante sulla performance per il momento!",
+  "lambdynlights.option.block_entities": "Entità blocco",
+  "lambdynlights.option.entities": "Entità",
+  "lambdynlights.option.mode": "Luci Dinamiche",
+  "lambdynlights.option.water_sensitive": "Controllo di sensibilità dell'acqua",
+  "lambdynlights.toast.first_time": "Disattivato di predefinito, controlla impostazioni della mod o di Grafica!",
```

Comment:
Il "disabled" si riferisce alla modifica, manca l'articolo, la preposizione è sbagliata e "Grafica" ha l'iniziale maiuscola.
In più credo possa essere riformulata per suonare meglio ed essere più accurata, visto che per impostazioni video credo si riferisca a quelle di Minecraft, separate da quelle della modifica.

"Disattivat**a** di predefinito, controlla **le** impostazioni **grafiche o** della mod~~o di Grafica~~!"

Reply:
il "Grafica" maiuscola si riferiva alle impostazioni del gioco che si chiamano giustappunto "Grafica", però hai ragione e lo cambio subito